Dockerfile

```diff
@@ -1,6 +1,6 @@
 # BUILD NIM APP ----------------------------------------------------------------
 
-FROM alpine:3.16 AS nim-build
+FROM alpine:edge AS nim-build
```

Comment:
Could we have problems in the future with this? Why not latest? Do we need the fixes?

Reply:
Added a comment in [4031a44](https://github.com/waku-org/nwaku/pull/1853/commits/4031a443c12737ee083e3cbd12f9583d44eb4382) to describe why we need edge. tl;dr, even while using 3.16 and installing the required toolchain, it still isn't able to resolve github dependencies